mods/d2k/chrome.yaml

```diff
@@ -97,7 +97,7 @@ flags: buttons.png
 	Random: 67,114,23,21
 	spectator: 67,114,23,21
 	corrino: 90,114,22,21
-	smuggler: 111,114,23,21
+	mercenary: 111,114,23,21
```

Comment:
Smugglers are a valid faction, just like the mercenaries.  This is just shifting the problem to a different owner!